docs/environment.md

```diff
@@ -404,6 +404,10 @@ Specify which authentication flow should be used when performing Microsoft authe
 
 Defaults to the value `auto`.
 
+**Note:** If [`GCM_MSAUTH_USEBROKER`](#gcm_msauth_usebroker) is set to `true`
+and the operating system authentication broker is available, all flows will be
+delegated to the broker; this setting has no effect.
```

Comment:
By "this setting" you mean `GCM_MSAUTH_FLOW`, correct? And by "has no effect" you mean to imply that `_USEBROKER` overrides `_FLOW`, correct?

Reply:
Same sentence suggestion as the other instance.